### PR TITLE
Added poppy to loadouts

### PR DESCRIPTION
**Pull request by foob1**
## About the PR
Added a poppy as a loadout option under trinkets.

## Why / Balance
Cosmetic reasons as the poppy can be used as a hairflower.

## Technical details
N/A

## Media
KtSLjir
1ljZDx9

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
N/A

**Changelog**

:cl:
- add: Added a poppy as a loadout item under trinkets.

### CODE REVIEW

**Comment by hord-brayden** [comment]
In b4 the comments about metagaming loadouts.

**Comment by foob1** [comment]
FYI everyone I'm trying this instead of #29302

**Comment by deltanedas** [comment]
time to chem powergame get free sutures without botany :trollface:

**Comment by thebadman4662** [comment]
> time to chem powergame get free sutures without botany :trollface:

First though unironically lmao. Thats 10 sutures per player. On some maps thats enough to set medical for whole shift.

Loadout/spawned poppies also have 1000%(or was it 500%?) chems so you can drain scoop them for 20 bic each.

**Comment by bruhmogus** [comment]
merge this asap PLEASE i need to look PRETTY lmao

**Comment by foob1** [comment]
> merge this asap PLEASE i need to look PRETTY lmao

so are you gonna give feedback or bash people for wanting to add content ingame

**Comment by Djungelskog2** [comment]
yea this is gonna be a powergaming issue

**Comment by foob1** [comment]
> yea this is gonna be a powergaming issue

Maintainers are alright with poppies being in autodrobes which are easily accessible to everyone on most maps (though I get it waiting for a maintainer to say something)

**Comment by Djungelskog2** [comment]
> > yea this is gonna be a powergaming issue
> 
> Maintainers are alright with poppies being in autodrobes which are easily accessible to everyone on most maps (though I get it waiting for a maintainer to say something)

theyre in theatre drobes p sure, and only have 1

**Comment by foob1** [comment]
> > > yea this is gonna be a powergaming issue
> 
> > 
> 
> > Maintainers are alright with poppies being in autodrobes which are easily accessible to everyone on most maps (though I get it waiting for a maintainer to say something)
> 
> 
> 
> theyre in theatre drobes p sure, and only have 1

https://github.com/space-wizards/space-station-14/pull/29302#issuecomment-2183032915

**Comment by hord-brayden** [comment]
FWIW, I don't think it'll be a huge issue, honestly. Currently in the trinket metagaming industry, it's lighter + cigs + cigar case for 16 ammunition storage, that can fit in a coat pocket, and the ability to make fire. My only concern would be immediate spawn meds. It's 20 bicaridine, so not a small amount of healing. But if people want to pull out a poppy to chomp on in the middle of a crowbar or gun fight - whatever. Not as crazy as spawning in with a 20dmg weapon, like the musician.

**Comment by IProduceWidgets** [comment]
I personally find all the metagamers wearing a hairpoppy to be hilarious and based.

**Comment by deltanedas** [comment]
new powergamer dogwhistle

**Comment by luizwritescode** [comment]
> Cosmetic reasons as the poppy can be used as a hairflower.

It really confuses me that a poppy being added for cosmetic reasons... is not purely cosmetic, and will probably be used for its reagents most of the time. On the other hand, the fake poppy was fine.

**Comment by Tayrtahn** [comment]
Heh, I think the best option might be a combination of both PRs: keep the real poppy in the drobe and add the fake poppy as a loadout item.

**Comment by a-person5660** [comment]
I don't think that people power gaming this is going to be very impactful or that big of a deal, I can forsee chemists and maybe doctors using this for a few extra sutures but not much more than that, maybe chewing some for a fight but you are better off using cloth or asking for chems frok med. It also means that they will have access to poppies when there is no botanists or on low pop. If there are botanists then poppies will be in abundance anyway so a handful will be a drop in a bucket. A few sutures will speed up med reviving dead people but they will likely still use chems for healing the living since its only two to five stacks of sutures from the medical staff if any.

TLDR: you guys are way over thinking the balance of this. It won't be that impactful and its cool so it should be merged.

**Comment by a-person5660** [comment]
Also a point based loadout system would literally solve all our problems.

**Comment by Ilya246** [comment]
get one poppy from loadout to chew after battle vs just ask botany for 20

**Comment by whateverusername0** [comment]
people arguing whether a roundstart poppy is powergaming is golden

**Comment by a-person5660** [comment]
I doubt powergamers would even think to do this for the free bic instead of ask botany for ambrosia deus (meth and omnizine)

**Comment by thebadman4662** [comment]
> people arguing whether a roundstart poppy is powergaming is golden

Its not because powergaming was removed from rules. :trollface:

**Comment by Killerqu00** [comment]
if balance of the game actually heavily depends on whether poppies can be added to loadouts or not, then the balance should be remade

**Comment by foob1** [comment]
Should I just reopen #29302 ? The only reason I made this PR was because emo said https://github.com/space-wizards/space-station-14/pull/29302#issuecomment-2183032915 and the PR hasn't been looked at since.

**Comment by Reese1243** [comment]
> if balance of the game actually heavily depends on whether poppies can be added to loadouts or not, then the balance should be remade

sometimes it feels like SS14 being balanced like a competitive shooter

**Review by LankLTE — CHANGES_REQUESTED** [review]
The poppy should be cosmetic if it's going to be in loadouts.

**Comment by a-person5660** [comment]
I would rather it be a real poppy as it gives a way for med to get them when there's no botany, and it's such a small amount that if there is botany then it's an inconsequential amount.

**Comment by foob1** [comment]
> The poppy should be cosmetic if it's going to be in loadouts. 

Alright closing this PR and reopening https://github.com/space-wizards/space-station-14/pull/29302

**Comment by foob1** [comment]
> The poppy should be cosmetic if it's going to be in loadouts. 

I'm a bit confused though because I was told this by another maintainer https://github.com/space-wizards/space-station-14/pull/29302#issuecomment-2183032915
